doc/adr/0005-wego-core-auth-strategy.md

```diff
@@ -4,7 +4,11 @@ Date: 2021-07-22
 
 ## Status
 
-Accepted
+~Accepted~
+
+Superseded by [ADR-0015 Current API Authentication Strategy](./0015-current-api-authentcation-stratgey.md)
+
+Authentication to Git, or git hosts is no longer supported
```

Comment:
I am still a bit unclear around this sentence, are we saying that from this ADR 

https://github.com/weaveworks/weave-gitops/pull/2581/files#diff-6dfa69fdfe6dd0e10e73325c13fc979e9359f1b00ca08468708d1a22271540b6R29 ?

Reply:
I think these are orthogonal issues, and they're being conflated here.

You need to be authenticated to Kubernetes (in some way) to query Kubernetes resources (and allow K8s RBAC to work).

You _may_ need to be authenticated to a Git hosting service e.g. GitHub, in order to query their API, for example, to change files and open a PR with the changes.

We will _always_ require K8s auth, we _may_ need you to authenticate to your Git hosting to carry out operations that require authentication there.

There are _two_ authentications.

Reply:
[ADR-0005](https://github.com/weaveworks/weave-gitops/blob/main/doc/adr/0005-wego-core-auth-strategy.md) discusses auth(Z|N) for 3 systems (git, the git-host and kubernetes). We actually add a 4th (authN with the gitops application itself).

The core thing I wanted to note here was that, of the original 3 from ADR-0005, gitops OSS only deals with one (authentication to kubernetes). ADR-0015 also adds some notes on how authentication to the gitops application is carried out (i.e. OIDC, cluster-user and token pass-through).

If you like I can add this additional detail to that & the context to make it clearer what the state of things is (this is why I initially was using statements like "partially superseded by" because ADR-0015 has relevance to ADR-0005 but isn't a direct descendent)

Reply:
I think #[9197190](https://github.com/weaveworks/weave-gitops/pull/2581/commits/919719004de2b35d23e5f4e09e6d1e8011163fec) should clarify this @enekofb

Reply:
Thanks for the effort @SamLR , I think it is not an issue of the ADR per se, it is just the need to reconstruct internal documentation out of ADRs that makes things a bit tricky 😅  ...